src/theme/styles/global.ts

```diff
@@ -65,33 +58,26 @@ export const StaticGlobalStyle = createGlobalStyle`
     height: 100vh;
     width: 100vw;
     margin: 0;
+    
     font-size: 62.5%;
-    line-height: 10px;
-    /* TODO: change the next 3 lines to be in themed global */
-    font-family: var(--font-default);
-    background-image: linear-gradient(0deg, var(--color-gradient-1) 0%, var(--color-gradient-2) 100%);
-    color: var(--color-text-primary);
-    /* End TODO */
-    box-sizing: border-box;
-    scroll-behavior: smooth;
     text-rendering: geometricPrecision;
+    line-height: 10px;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+    
+    box-sizing: border-box;
+    
+    overscroll-behavior-y: none;
+    scroll-behavior: smooth;
   }
 
   *, *:before, *:after {
     box-sizing: inherit;
   }
 
-  a {   
-    text-decoration: underline;
-    cursor: pointer;
-      &:link, 
-      &:visited {
-        color: var(--color-text-active);
-      }
-  }
+  
 
+  /* TODO: move closer to H elements or sth */
```

Comment:
why? imo these styles that are not themed should be sth similar to a CSS reset imo:
https://meyerweb.com/eric/tools/css/reset/